internal/provider/resource_project.go

```diff
@@ -29,12 +29,6 @@ func resourceProject() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
-			"description": {
-				Description: "Project description",
```

Comment:
Removal of this resource will cause issues for anyone using the provider, the recommended best practice is to mark it was **Deprecated** and then remove it in the next major reversion.

https://www.terraform.io/plugin/sdkv2/best-practices/deprecations#provider-attribute-removal

Reply:
Agreed, thank you @jjm for pointing this out. We will follow best practices and first mark the field as deprecated in this version, removing it in the next major version.